components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
@mike-robertson should we adopt this for the rest of the components as well?  Current table don't show up well on mobile.